android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerViewManager.java

```diff
@@ -211,14 +213,14 @@ public void setSrc(final ReactExoplayerView videoView, @Nullable ReadableMap src
         } else {
             int identifier = context.getResources().getIdentifier(
                 uriString,
-                "drawable",
-                context.getPackageName()
+                    "drawable",
+                    context.getPackageName()
```

Comment:
```suggestion
                context.getPackageName()
```

Reply:
chore: revert indent change
```suggestion
                context.getPackageName()
```